### ISSUE TITLE
PERFORMANCE: Make it possible to specify number of cores

### ISSUE THREAD

**Issue by HenrikBengtsson** [UCSF-Costello-Lab/LG3_Pipeline#46]
We currently have hardcoded the number of parallel tasks in the header of the PBS scripts, e.g.
```
#PBS -l nodes=1:bigmem:ppn=12,vmem=100gb
```
Note, we've already fixed the scripts such that they automatically acknowledge these assigned cores; see Issue #14.  So that part is already taken care of.

First, any `#PBS ` directives in a PBS file are overridden if also specified as `qsub` options.  For example, calling the above PBS script as:
```
$ qsub -l nodes=1:bigmem:ppn=24 foo.pbs
```
will use 24 cores instead of the default 12.  I've just verified that we still get the default `vmem=100gb`.

This means that we can keep the default `#PBS ...` directives and provide a mean to override these from the run scripts, e.g. via env variables (or the future config file).

PS. This would allow us to speed up the testing of certain multi-hour steps.

**Comment by HenrikBengtsson** [comment]
I added a very first rudimentary support for this.  I'm right now running the Recal step with 48 cores (instead of the default 12) by launching as:
```sh
$ QSUB_OPTS="-l nodes=1:ppn=48" ./_run_Recal
```
I've updated all the run scripts such that one can pass an initial value of `QSUB_OPTS` (which is then appended to).  However, I don't think this will work for the run scripts that calls scripts that in turn launched PBS scripts.
